### PR TITLE
Added hero image dimensions to  _template-post

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

**Comment by samdutton** [comment]
BTW: Are the suggested thumbnail dimensions correct? 

The maximum dimensions on desktop seem to be 365 x 240, so maybe the recommendation here should be 730 x 480? Mobile dimensions (in DevTools, at least) are 363 to 366 x 240 (depending on the device).

**Comment by kaycebasques** [comment]
@robdodson Sam has a question about thumbnail dimensions

**Comment by robdodson** [comment]
Good question. The thumbnails are (unfortunately) fluid so at their largest they're 448 x 240 (at their smallest they're around 330 x 220). But yeah, 2x that would mean 896 width.

**Comment by kaycebasques** [comment]
@samdutton 896 width by 480 height seems like good dimensions to specify for the thumbnails in that case

File: src/site/_drafts/_template-post/index.md [hunk 1]
```diff
@@ -38,7 +39,7 @@ hero: hero.jpg
 alt: A description of the hero image for screen reader users.
 
 # You can provide a custom thumbnail and description for social media cards.
-# Images should be 1200 x 630.
+# Thumbnail images should be 1200 x 630.
```
Comment (kaycebasques) [inline src/site/_drafts/_template-post/index.md:42]:
```suggestion
# Thumbnail images should be 896 x 480.
```

**Pull request merged**